From 97289b56bdf09be9f2e49c196c666bd41185c2ca Mon Sep 17 00:00:00 2001
From: Mika Laitio <lamikr@gmail.com>
Date: Fri, 5 Jul 2024 12:32:20 -0400
Subject: [PATCH 3/4] remove HIP version from search

HIP 1.0 could never be found now when
rocm/HIP version is already 6.1

Signed-off-by: Mika Laitio <lamikr@gmail.com>
---
 cmake/LoadHIP.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake/LoadHIP.cmake b/cmake/LoadHIP.cmake
index 09ae6385..4f695e01 100644
--- a/cmake/LoadHIP.cmake
+++ b/cmake/LoadHIP.cmake
@@ -140,7 +140,7 @@ macro(find_package_and_print_version PACKAGE_NAME)
 endmacro()
 
 # Find the HIP Package
-find_package_and_print_version(HIP 1.0)
+find_package_and_print_version(HIP)
 
 if(HIP_FOUND)
   set(PYTORCH_FOUND_HIP TRUE)
-- 
2.46.0

